engine/graphics/src/webgpu/graphics_webgpu.cpp

```diff
@@ -281,13 +299,20 @@ static size_t WebGPUCompressedBlockWidth(TextureFormat format)
     assert(format <= TEXTURE_FORMAT_COUNT);
     switch (format)
     {
-    case TEXTURE_FORMAT_RGB_ETC1:           return 4;
-    case TEXTURE_FORMAT_RGBA_ETC2:          return 4;
-    case TEXTURE_FORMAT_RGBA_ASTC_4x4:      return 4;
-    case TEXTURE_FORMAT_RGB_BC1:            return 4;
-    case TEXTURE_FORMAT_RGBA_BC3:           return 4;
-    case TEXTURE_FORMAT_RGBA_BC7:           return 4;
-    default:                                return 0;
+        case TEXTURE_FORMAT_RGB_ETC1:
+            return 4;
+        case TEXTURE_FORMAT_RGBA_ETC2:
+            return 4;
+        case TEXTURE_FORMAT_RGBA_ASTC_4x4:
+            return 4;
+        case TEXTURE_FORMAT_RGB_BC1:
+            return 4;
+        case TEXTURE_FORMAT_RGBA_BC3:
+            return 4;
+        case TEXTURE_FORMAT_RGBA_BC7:
+            return 4;
+        default:
+            return 0;
```

Comment:
I think our clang-format strikes again.
I kind of like the readability of the previous code better 🤔